src/envs/cover.py

```diff
@@ -1073,18 +1073,8 @@ def _Pick_learned_equivalent_terminal(self, s: State, m: Dict,
                                           o: Sequence[Object],
                                           p: Array) -> bool:
```

Comment:
how is this function different from Pick_terminal above? i see that `robot` is an extra argument but i'm unsure why

Reply:
this is related to your question in the other PR. this parameterized option should match the one that we anticipate learning exactly.